### ISSUE TITLE
Any chance for chroot?

### ISSUE THREAD

**Issue by rzmatlr** [xoureldeen/Vectras-VM-Android#136]
I realize this project uses proot.
Proot is slow in my using, and I feel chroot would be a useful addition for us with rooted devices.
Would you consider adding chroot?

**Comment by xoureldeen** [comment]
> I realize this project uses proot. Proot is slow in my using, and I feel chroot would be a useful addition for us with rooted devices. Would you consider adding chroot?

yes i looking for, but currently I don't have rooted device for testing.

**Comment by rooteduniverse1003** [comment]
How do you not have a rooted device???  It's not hard at all!! But I have a Moto G Stylus 5G 2024 n ur app works great on it! I wish ur app had support for Gunyah Hypervisor n KVM!!!

**Comment by xoureldeen** [comment]
> How do you not have a rooted device??? It's not hard at all!! But I have a Moto G Stylus 5G 2024 n ur app works great on it! I wish ur app had support for Gunyah Hypervisor n KVM!!!

yes i know how to root my device but i don't have time, and currently I don't want to lose any data because im at school currently, thank you!
